src/system.c

```
@@ -287,6 +293,36 @@ void sysReset()
     poweroffSetCallback(&poweroffHandler, NULL);
 }
 
+// Deferred halves of the old sysReset module set (see the comment there). Both are idempotent
+// one-shots that latch ONLY on full success (a transient load failure stays retryable -- the exact
+// latch-on-failure class the 2026-07 atad/mmce fixes eradicated; CodeRabbit review of #252), and
+// sysReset() clears the latches so an in-process IOP reboot reloads everything. Audio pair =
+// deferredAudioInit's first act; launch pair = deferredInit (IO worker, GUI path) or the autolaunch
+// fork (synchronous -- it skips deferredInit).
+void sysLoadAudioModules(void)
```

Cppcheck (2.21.0)

[style] The function 'sysLoadAudioModules' is never used.

(unusedFunction)